src/testing/jest/jest-27-and-under/jest-config.ts

```diff
@@ -146,7 +145,6 @@ export function buildJestConfig(config: d.ValidatedConfig): string {
     jestConfig.verbose = stencilConfigTesting.verbose;
   }
 
-  // TODO(STENCIL-307): Move away from Jasmine runner for Stencil tests, which involves a potentially breaking change
   jestConfig.testRunner = 'jest-jasmine2';
```

Comment:
Technically, Jest 27 supports jest-jasmine2 and jest-circus. At this time, we don't allow folks on Jest 27 and lower to configure this. We may need to change that in the future, but I believe the changes would be more than updating the assignment here to 'jest-circus' or something like:
```ts
jestConfig.testRunner = stencilConfigTesting.testRunner ?? 'jest-jasmine2';
```